contracts/VaultV3.vy

```diff
@@ -968,11 +982,13 @@ def approve(spender: address, amount: uint256) -> bool:
 
 @external
 def transfer(receiver: address, amount: uint256) -> bool:
+    self.is_whitelisted(receiver)
```

Comment:
same as above, a "blackListed" address already in the vault can transfer itself out as long as he uses a diff receiver, not sure if the feature is meant to lock funds, but would need check on both addresses IMO

Reply:
yeap, the more I think of it, the more I get convinced that msg.sender should also be included